package.json

```diff
@@ -115,5 +115,6 @@
   },
   "dependencies": {
     "upgrade": "1.1.0"
-  }
+  },
+  "version": ""
```

Comment:
whatever meh

Reply:
should we make it 0.0.0 for consistency and convention as internal pkg?

Reply:
Mh yeah, why not.
```suggestion
  "version": "0.0.0"
```